service/src/main/java/org/jboss/sbomer/service/feature/sbom/model/Stats.java

```diff
@@ -51,7 +61,8 @@ public static class Producer {
     @Data
     @Builder(setterPrefix = "with")
     public static class Messaging {
-        Consumer consumer;
+        PncConsumer pncConsumer;
+        ErrataConsumer errataConsumer;
```

Comment:
We need to update UI to reflect this change.

Reply:
Ah right! Thanks, updated!